src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_table/table_grid.tsx

```diff
@@ -240,7 +238,7 @@ export function TableGrid({
         defaultMessage: 'Field values',
       })}
       className="kbnDocViewer__fieldsGrid"
-      css={[styles.fieldsGrid, hideDataGridHeader && styles.hideDataGridHeader]}
+      css={[styles.fieldsGrid]}
```

Comment:
```suggestion
      css={styles.fieldsGrid}
```
Nit: should be able to pass this directly now.

Reply:
https://github.com/elastic/kibana/pull/233661/commits/fc665dcdbf505aaeb72e88c5a0a37d10ba395d97